test/functional/sqlsrv/sqlsrv_prepare.phpt

```diff
@@ -121,14 +129,14 @@ binding parameters, including output parameters, using the simplified syntax.
     sqlsrv_free_stmt($stmt);
     sqlsrv_close($conn);
 ?>
---EXPECTF--
+--EXPECTREGEX--
 1
-12.0
+12\.0
 testtestte
-This is some text meant to test binding parameters to streams
+(This is some text meant to test binding parameters to streams)?
```

Comment:
should compare the results in the test instead (what if column encryption is enabled and it successfully fetches the stream, the test would still pass)